test/test_message.cpp

```diff
@@ -4148,6 +4148,7 @@ BOOST_AUTO_TEST_CASE(parse_multipart_content)
         "--my_bound\r\n"
         "Content-Type: text/html; charset=utf-8\r\n"
         "Content-Transfer-Encoding: Base64\r\n"
+        "Content-Disposition: form-data; name=\"name\"\r\n"
```

Comment:
There is no `form-data` in the Content Disposition description in the RFC 2183. I would say it is related only to the HTTP specification. However, there is a MIME multipart subtype `form-data` which could be used for these purposes.